go/arrow/internal/arrjson/arrjson.go

```diff
@@ -52,15 +52,16 @@ type Field struct {
 }
 
 type dataType struct {
-	Name      string `json:"name"`
-	Signed    bool   `json:"isSigned,omitempty"`
-	BitWidth  int    `json:"bitWidth,omitempty"`
-	Precision string `json:"precision,omitempty"`
-	ByteWidth int    `json:"byteWidth,omitempty"`
-	ListSize  int32  `json:"listSize,omitempty"`
-	Unit      string `json:"unit,omitempty"`
-	TimeZone  string `json:"timezone,omitempty"`
-	Scale     int    `json:"scale,omitempty"` // for Decimal128
+	Name       string `json:"name"`
```

Comment:
this is just whitespace adjustment?

Reply:
`KeysSorted` was added to the struct which is why the whitespace got adjusted